Makefile

```
@@ -85,6 +85,11 @@ install-samples: export KUBECONFIG = $(KIND_KUBECONFIG)
 install-samples: generate-go install-crd ## Install samples into cluster
 	yq package/samples/*.yaml | kubectl apply -f -
 
+.PHONY: delete-instance
+delete-instance:  ## Deletes sample instance if it exists
+	yq '.metadata.name' package/samples/postgresql.appcat.vshn.io_postgresqlstandalone.yaml | \
+ 	xargs kubectl delete postgresqlstandalone --ignore-not-found=true
```

I believe `delete` also accepts a filename if I'm not mistaken.
```suggestion
 	kubectl delete -f package/samples/postgresql.appcat.vshn.io_postgresqlstandalone.yaml --ignore-not-found=true
```